steam_deploy.sh

```diff
@@ -56,13 +56,23 @@ echo ""
 
 mkdir BuildOutput
 
+steamdir=$STEAM_HOME
+manifest_path=$(pwd)/manifest.vdf
+contentroot=$(pwd)/$rootPath
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  steamdir="$HOME/Library/Application Support/Steam"
```

Comment:
Honestly, seems like a bug in Steam that steamcmd clobbers your main Steam app authentication.